spec/nested-rules.js

```diff
@@ -0,0 +1,90 @@
+if (typeof require !== 'undefined') {
+    var Validator = require('../src/validator.js');
+    var expect = require('chai').expect;
+} else {
+    var Validator = window.Validator;
+    var expect = window.chai.expect;
+}
+
+describe('nested validation rules', function() {
+    var nestedObject = {
+        name: 'required',
+        data: {
+            weight: 'required',
+            hair: {
+                color: 'required'
+            }
+        }
+    };
+
+    var nestedFlatten = {
+        name: 'required',
+        'data.weight': 'required',
+        'data.hair.color': 'required'
+    };
+
+    var dataPass = {
+        name: 'David',
+        data: {
+            weight: 70,
+            hair: {
+                color: 'black'
+            }
+        }
+    }
```

Comment:
please add semicolon. also, this looks like 4 spaces. could you change it all to 2 spaces?